provider/gradle-mvn-push.gradle

```diff
@@ -105,19 +109,4 @@ afterEvaluate { project ->
         required { isReleaseBuild() }
         sign publishing.publications.mavenJava
     }
-
-    task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
-        archiveClassifier = 'javadoc'
-        from androidJavadocs.destinationDir
-    }
-
-    task androidSourcesJar(type: Jar) {
-        archiveClassifier = 'sources'
-        from android.sourceSets.main.java.sourceFiles
-    }
-
-    artifacts {
-        archives androidSourcesJar
-        archives androidJavadocsJar
-    }
```

Comment:
The following tasks are not used, have been replaced by the `android.publishing` definition